### ISSUE TITLE
[regression] `no-duplicates` false positive

### ISSUE THREAD

**Issue by frantic1048** [import-js/eslint-plugin-import#2934]
With latest version of `eslint-plugin-import`: `2.29.0`   
and ESLint rule `'import/no-duplicates': ['error', { 'prefer-inline': true }]`

Following TypeScript code triggers `no-duplicates` error:

```ts
import type Highcharts from 'highcharts'
import { type SankeyNodeObject } from 'highcharts'
```

**Expected:**

no error.

## Reference

Similar issue fixed in the past: 

* https://github.com/import-js/eslint-plugin-import/issues/2114

**Comment by frantic1048** [comment]
I apologize for any inconvenience caused. I misinterpreted the `prefer-inline` options during testing. Therefore, this issue is not valid.
